doc/content/lorawan/regional-limitations-of-rf-use/_index.md

```diff
@@ -0,0 +1,65 @@
+- M (868 MHz - 868.6 MHz): 1%, 25 mW ERP
+- N (868.7 MHz - 869.2 MHz): 0.1%, 25 mW ERP
+- P (869.4 MHz - 869.65 MHz): 10%, 500 mW ERP
+- Q (869.7 MHz - 870 MHz): 1%, 25 mW ERP
+
+For example, devices operating in the 869,4-869,65 MHz range, which is sub-band P, have a duty cycle limitation of 10%, which means that a device can transmit for no more than 10% of the time.
+
+Usually, the limitations of sub-band K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. LoRaWAN's RX2 channel is mainly used for downlink communication because there are more end devices with individual duty cycles compared to gateways, and the RX2 channel is in a specific frequency sub-band for gateways to handle the imbalance between end devices and gateways.
+
+By default, the RX1 receive window for downlinks uses the same frequency as the preceding uplink, while the RX2 receive window uses a fixed frequency, with the default being 869.525 MHz at DR 0 using SF12 with 125 kHz.
+
+The default frequency for beacon broadcasts and ping-slot transmissions for Class B end devices in Europe is 869.525 MHz. For LoRaWAN relays, channel 0 uses 865.1 MHz and 865.5 MHz for WOR (Wake On Radio) transmissions respectively, and channel 1 uses 865.3 MHz and 865.9 MHz for WOR ACK (Wake On Radio Acknowledgement) transmissions respectively.
+
+# US (902-928 MHz)
+
+The following regulations are applied to the USA, Canada, and all other countries in ITU Region 2 adopting the entire [FCC](https://www.fcc.gov/) (Federal Communications Commission) 47 CFR Part 15 regulations in the 902-928 ISM band.
+
+In the United States, the ISM band is divided into **80 channels**: 72 for uplink and 8 for downlink.
+
+- There are **64 uplink channels**, numbered from 0 to 63, each with a bandwidth of 125 kHz. These channels start at a center frequency at 902.3 MHz and increment linearly by 200 kHz up to 914.9 MHz.
+- There are **8 uplink channels**, numbered from 64 to 71, each with a bandwidth of 500 kHz. These channels start at a center frequency at 903.0 MHz and increment linearly by 1.6 MHz up to 914.2 MHz.
+- There are **8 downlink channels**, numbered from 0 to 7, each with a bandwidth of 500 kHz. These channels start at a center frequency at 923.3 MHz and increment linearly by 600 kHz up to 927.5 MHz.
+
+The end devices must have the capability to store parameters for the 72 uplink channels mentioned above.
+
+The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that the devices are able to successfully transmit data while avoiding interference with other devices.
+
+The value of dwell time for LoRaWAN for the US902-928 band is 400ms. This means that the device should spend 400 ms on each channel before switching to the next channel.
```

Comment:
> should spend 400 ms on each channel

I interpret the rules differently. When I google most people says something like
> dwell time shall not exceed 400 mS

So in other words, we are not required to stay the full 400ms on the channel. We can stay there shorter and than hop to another frequency. So using the words "should spend" is incorrect.